lib/gltfPipeline.js

```diff
@@ -26,13 +29,24 @@ module.exports = {
     processFileToDisk : processFileToDisk
 };
 
-function processJSON(gltf, options, callback) {
+function writeSources(gltf, outputPath, options, callback) {
+    var embed = options.embed;
+    var embedImage = options.embedImage;
+    var basePath = path.dirname(outputPath);
+    async.each(['buffers', 'images', 'shaders'], function(name) {
+        writeSource(gltf, basePath, name, embed, embedImage, callback);
+    });
+}
+
+function processJSON(gltf, outputPath, options, callback) {
```

Comment:
I may have misunderstood how we wanted to resolve this issue. I thought we wanted the capability to output non-embedded files when using `processJSON` and `processFile`. If we are fine with them always being embedded, then that does simplify the solution (RE: your comment below - the only reason I refactored `processJSONToDisk` and `processFileToDisk` was to allow non-embedded).